components/accordion/metadata/mods.md

```diff
@@ -1,4 +1,4 @@
-| Modifiable Custom Properties                          |
+| Modifiable custom properties                          |
```

Comment:
This is super minor and a tiny bit annoying but the content style guide for the site requires sentence case headings so this gets us ready to publish this content on the pages.